github/GithubIntegration.py

```diff
@@ -26,6 +29,12 @@ def __init__(
         integration_id=None,
         private_key=None,
         base_url=Consts.DEFAULT_BASE_URL,
+        timeout=Consts.DEFAULT_TIMEOUT,
+        user_agent=Consts.DEFAULT_USER_AGENT,
+        per_page=Consts.DEFAULT_PER_PAGE,
+        verify=True,
+        retry=None,
+        pool_size=None,
```

Comment:
This is an API breaking change if end users are using positional arguments

Reply:
That is right. The rational here is the `jwt_*` arguments have been added quite recently and should be rarely used. So this breaks for little user code only, and is easy to fix. This will be flagged in the change log.

I favour similarity to `github.Github` signature here over not breaking this API for everyone.